scss/base/_mixins.scss

```
@@ -0,0 +1,60 @@
+@mixin flex($justify, $align, $direction: row, $wrap: nowrap, $gap: 0) {
+  display: flex;
+  justify-content: $justify;
+  align-items: $align;
+  @if ($direction != 'row') {
+    flex-direction: $direction;
+  }
+  @if ($wrap != 'nowrap') {
+    flex-wrap: $wrap;
+  }
+  @if ($gap != 0) {
+    gap: $gap;
+  }
+}
+@mixin abs-center($direction: 'both') {
+  position: absolute;
+  @if ($direction == 'x') {
+    left: 50%;
+    transform: translateX(-50%);
+  }
+  @if ($direction == 'y') {
+    top: 50%;
+    transform: translateY(-50%);
+  } @else {
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+  }
+}
+
+$breakpoints: (
+  'mobile': (
```

desktop first thì
```suggestion
  'mobile-and-down': (
```
mobile first thì ngược lại `up`